app/controllers/videos_controller.rb

```diff
@@ -1,6 +1,7 @@
 class VideosController < ApplicationController
   def index
     @categories = Category.all
+    @videos_without_categories = Video.where(category_id: nil)
```

Comment:
This kind of logic could be pushed model and build a scope off it. Read more about scopes [here](http://guides.rubyonrails.org/active_record_querying.html#scopes)